### PR TITLE
fix: offline load data & show table status

### LINKED ISSUE #3342: show table status show symbolic path
https://github.com/4paradigm/OpenMLDB/blob/63599455cdd22e2bdf84291838bbdc55b7e85766/src/sdk/sql_cluster_router.cc#L4169-L4174

only deep copy path, if user only soft copy, empty here, weird

### PR DESCRIPTION
**Pull request by vagetablechicken**
closes #3342 and hive load fix

### CODE REVIEW

File: src/sdk/sql_cluster_router.cc [hunk 1]
```diff
@@ -4068,7 +4067,7 @@ static const std::initializer_list<std::string> GetTableStatusSchema() {
     static const std::initializer_list<std::string> schema = {
         "Table_id",         "Table_name",     "Database_name",     "Storage_type",      "Rows",
         "Memory_data_size", "Disk_data_size", "Partition",         "Partition_unalive", "Replica",
-        "Offline_path",     "Offline_format", "Offline_deep_copy", "Warnings"};
+        "Offline_path",     "Offline_format", "Offline_symbolic_paths", "Warnings"};
```
Comment (aceforeverd) [inline src/sdk/sql_cluster_router.cc]: delete `deep_copy` ? SQL semantic breaks.

Comment (vagetablechicken) [reply src/sdk/sql_cluster_router.cc]: deep copy is an option in load data, not a needed field in table info. Actually, in offline table info proto, deep_copy is deprecated.

File: src/sdk/sql_cluster_router.cc [hunk 2]
```diff
@@ -1970,10 +1970,9 @@ base::Status SQLClusterRouter::HandleSQLCreateTable(hybridse::node::CreatePlanNo
 
         std::map<std::string, std::string> config;
         ::openmldb::taskmanager::JobInfo job_info;
-        int job_timeout = GetJobTimeout();
         std::string output;
-
-        ::openmldb::base::Status status = ExecuteOfflineQueryGetOutput(sql, config, db, job_timeout, &output);
+        // create table like hive is a long op, use large timeout
+        ::openmldb::base::Status status = ExecuteOfflineQueryGetOutput(sql, config, db, FLAGS_sync_job_timeout, &output);
```
Comment (vagetablechicken) [inline src/sdk/sql_cluster_router.cc]: cpplint

File: docs/zh/openmldb_sql/ddl/SHOW_TABLE_STATUS.md [hunk 3]
```diff
@@ -9,22 +9,22 @@ SHOW TABLE STATUS [LIKE Pattern];
...
+| Offline_format         | 表对应 offline 数据格式，仅对离线表生效。 `NULL`  表示未设置该项。                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| Offline_symbolic_paths | 表对应 offline 软链接路径，`NULL` 表示未设置该项。                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (vagetablechicken) [inline docs/zh/openmldb_sql/ddl/SHOW_TABLE_STATUS.md:26]: add a note about old 'deep_copy' field, add an issue for changelog

**Review by dl239 — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**